Automattic/jetpack
# Update/view summary cta text

Update 'View Summary' button text

#### Changes proposed in this Pull Request:
This updates the 'View Summary' on the recommendation prompts to 'View Recommendations' to make it more clear where it goes when it is clicked

#### Other information:


#### Jetpack product discussion
P2: p8oabR-Vp-p2#comment-6571

#### Does this pull request change what data or activity we track or use?
Nope

#### Testing instructions:
* Check out this branch via `git switch update/view-summary-cta-text`
* Spin up your local environment
* Head to any of the recommendations. For example: `/wp-admin/admin.php?page=jetpack#/recommendations/related-posts`
* Confirm that the `View Summary` text has been changed to `View Recommendations`
* Click on `View Recommendations` to confirm it still goes to the Assistant summary page
* Check other recommendations if you're not convinced 😄
